### ISSUE TITLE
How to test API alone

### ISSUE THREAD

**Issue by newkillerbeast2017** [jaesung2061/anvel#24]
Hi, how can I test the API alone? Is there a way I can access the API routes in postman to test it and then build the angular part?

**Comment by newkillerbeast2017** [comment]
@jaesung2061 any help?

**Comment by jaesung2061** [comment]
Can you rephrase that question for me?

**Comment by newkillerbeast2017** [comment]
@jaesung2061 I want to use your project as a base for mine. I dont know angular but my team mate does. For him to start, it may take a while so until then I want to complete a few routes and test them using postman. But the existing route you have, I cant access them via postman. Is there a way I can test the APIs alone without using angular or doing any work on the frontend?

**Comment by jaesung2061** [comment]
Sorry for the late response. Try adding `API_STRICT=false` to your .env file.

**Comment by newkillerbeast2017** [comment]
@jaesung2061 works now. Thanks! Just so that you know, AuthController@destroy doesn't exist.

**Comment by jaesung2061** [comment]
Got it, thanks!

**Comment by newkillerbeast2017** [comment]
@jaesung2061 will it updated to lumen 5.4?

**Comment by jaesung2061** [comment]
Yes. I will update it this weekend if I have time.
